content/en/network_monitoring/performance/setup.md

```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
+[The Datadog Operator][1] is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
+
+1. To enable Network Performance Monitoring in Operator, use the following configuration:
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: placeholder
+  namespace: placeholder
+spec:
+  # (...)
+  features:
+    networkMonitoring:
+      enabled: true
+  agent:
+    # (...)
+    systemProbe:
+      enabled: true
```

Comment:
I think you also need to set for NPM to work properly:
```
agent:
  process:
    enabled: true
```

Reply:
It should not be the case.

with:
```yaml
 features:
    networkMonitoring:
      enabled: true
```
the system-probe container will be enable automatically.

Reply:
@clamoriniere is correct. There is some glue code that makes this work. You only need what he said.